pkgs/top-level/perl-packages.nix

```diff
@@ -210,6 +210,31 @@ let
      };
   };
 
+  AlienSDL = buildPerlModule {
+    pname = "Alien-SDL";
+    version = "1.446";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/F/FR/FROGGS/Alien-SDL-1.446.tar.gz";
+      sha256 = "c9aa2c9dc3c63d89773c7d7203f2a46d1b924d0c72d9f801af147a3dc8bc512a";
+    };
+    patches = [ ../development/perl-modules/alien-sdl.patch ];
+
+    installPhase = ''
+      runHook preInstall
+      ./Build install --prefix $out
+      runHook postInstall
+    '';
+
+    buildInputs = [ ArchiveExtract ArchiveZip ModuleBuild TextPatch pkgs.SDL ];
+    SDL_INST_DIR = pkgs.SDL.dev;
```

Comment:
It builds the same module without `SDL_INST_DIR = pkgs.SDL.dev;`
`pkgs.SDL` in `buildInputs` seems enough to locate SDL

Reply:
It does compile, but the config will contain `sdl-config` instead of `/nix/store/XXXXX/bin/sdl-config`, has been my experience, I'll double check if that's still the case.

Reply:
Yeah, `(perlPackages.AlienSDL)/lib/perl5/site_perl/5.30.0/Alien/SDL/ConfigData.pm` has `'script' => 'sdl-config'` instead of a full path.